### ISSUE TITLE
does not take PBG URL other than http://download.geofabrik.de

### ISSUE THREAD

**Issue by pdhuvad** [peter-evans/nominatim-docker#15]
I am trying to install planet osm database. It does not take any other url other than one from  http://download.geofabrik.de. 
I get following error
`sudo docker run -d -p 8080:8080 -e NOMINATIM_PBF_URL='https://planet.openstreetmap.org/pbf/planet-latest.osm.pbf' --name nominatim peterevans/nominatim:latest`
```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (77) Problem with the SSL CA cert (path? access rights?)
 * Starting PostgreSQL 9.5 database server
   ...done.
NOTICE:  database "nominatim" does not exist, skipping
```

Thanks.

**Comment by peter-evans** [comment]
Hi @pdhuvad 

I added the package `ca-certificates` and it may have fixed the issue.
It's difficult for me to test because the planet pbf is so large.
Please could you try this release candidate and let me know if it works for you.

``` bash
docker run -d -p 8080:8080 \
-e NOMINATIM_PBF_URL='https://planet.openstreetmap.org/pbf/planet-latest.osm.pbf' \
--name nominatim peterevans/nominatim:1.3.4-rc.1
```

**Comment by peter-evans** [comment]
I've release this fix in version 1.3.4 so I'll close this issue. If you still have the same problem with the latest version let me know.
